core/src/test/scala/integration/kafka/api/DescribeAuthorizedOperationsTest.scala

```diff
@@ -1,210 +0,0 @@
-/**
```

Comment:
Do we have another test that validates this with KRaft?

Reply:
No, we don't have. I disable this test now. I prefer to create another Jira for followup. What do you think?